nexus/src/db/model.rs

```diff
@@ -1234,8 +1284,7 @@ impl Vpc {
             None => defaults::random_unique_local_ipv6(),
             Some(prefix) => {
                 // TODO: Delegate to `Ipv6Addr::is_unique_local()` when stabilized.
```

Comment:
So, I added that one in the previous changes. I still think it stands, since the `is_unique_local()` that we're calling is not the one on `std::net::Ipv6Addr`, but the [one on our wrapper type](https://github.com/oxidecomputer/omicron/blob/0299cbfc13d77662e752848e5a8715505986602f/common/src/api/external/mod.rs#L1113). That might be confusing, and perhaps I should rename that method. But I do think the TODO is still helpful! Or maybe I should just attach it to that method, since that's what should ultimately delegate?

Reply:
I've moved this comment over to the internals of the method on our wrapper type.